sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -1449,19 +1449,13 @@ class MLPRegressor(RegressorMixin, BaseMultilayerPerceptron):
     out_activation_ : str
         Name of the output activation function.
 
-    Examples
+    See Also
     --------
-    >>> from sklearn.neural_network import MLPRegressor
-    >>> from sklearn.datasets import make_regression
-    >>> from sklearn.model_selection import train_test_split
-    >>> X, y = make_regression(n_samples=200, random_state=1)
-    >>> X_train, X_test, y_train, y_test = train_test_split(X, y,
-    ...                                                     random_state=1)
-    >>> regr = MLPRegressor(random_state=1, max_iter=500).fit(X_train, y_train)
-    >>> regr.predict(X_test[:2])
-    array([-0.9..., -7.1...])
-    >>> regr.score(X_test, y_test)
-    0.4...
+    AdamOptimizer : Stochastic gradient descent optimizer with Adam.
+    BaseOptimizer : Base (Stochastic) gradient descent optimizer.
+    BernoulliRBM : Bernoulli Restricted Boltzmann Machine (RBM).
+    MLPClassifier : Multi-layer Perceptron classifier.
+    SGDOptimizer : Stochastic gradient descent optimizer with momentum.
```

Comment:
We don't have a class `SGDOptimizer`. However, you can link to the `SGDRegressor`: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDRegressor.html